src/kvstore/test/NebulaStoreTest.cpp

```diff
@@ -825,6 +826,100 @@ TEST(NebulaStoreTest, ThreeCopiesCheckpointTest) {
     }
 }
 
+TEST(NebulaStoreTest, AtomicOpBatchTest) {
+    auto partMan = std::make_unique<MemPartManager>();
+    auto ioThreadPool = std::make_shared<folly::IOThreadPoolExecutor>(4);
+    // space id : 1 , part id : 0
+    partMan->partsMap_[1][0] = PartMeta();
+
+    VLOG(1) << "Total space num is " << partMan->partsMap_.size()
+            << ", total local partitions num is "
+            << partMan->parts(HostAddr(0, 0)).size();
+
+    fs::TempDir rootPath("/tmp/nebula_store_test.XXXXXX");
+    std::vector<std::string> paths;
+    paths.emplace_back(folly::stringPrintf("%s/disk1", rootPath.path()));
+
+    KVOptions options;
+    options.dataPaths_ = std::move(paths);
+    options.partMan_ = std::move(partMan);
+    HostAddr local = {0, 0};
+    auto store = std::make_unique<NebulaStore>(std::move(options),
+                                               ioThreadPool,
+                                               local,
+                                               getHandlers());
+    store->init();
+    sleep(FLAGS_raft_heartbeat_interval_secs);
+    // put kv
+    {
+        std::vector<std::pair<std::string, std::string>> expected, result;
+        auto atomic = [&]() -> std::string {
+            std::unique_ptr<kvstore::BatchHolder> batchHolder =
+                    std::make_unique<kvstore::BatchHolder>();
+            for (auto i = 0; i < 20; i++) {
+                auto key = folly::stringPrintf("key_%d", i);
+                auto val = folly::stringPrintf("val_%d", i);
+                batchHolder->put(key, val);
```

Comment:
As your usage,  i suggest we use std::string as the argument type for batchHolder->put